### PR TITLE
Update Chinese Strings

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
+    <string name="shared_pref_key_isIconHidden" translatable="false">isIconHidden</string>
+    <string name="string_all_key" translatable="false">all.all.all.all.all</string>
+
+    <string name="string_filter_intent" translatable="false">com.iboalali.sysnotifsnooze.NOTIFICATION_LISTENER_SERVICE_INTENT</string>
+    <string name="shared_pref_key_version_code" translatable="false">version_code</string>
 
     <!-- To all contributors, Please do not translate these two. I will use the translation used by AOSP.
             If there are any questions or change requests about these translations please contact me -->
-    <string name="string_app_name_replace_running_in_the_background">正在后台运行</string>
-    <string name="string_app_name_replace_using_battery">正在消耗电量</string>
```
Comment (iboalali) [inline app/src/main/res/values-zh-rCN/strings.xml:39]: These two should not be translated by anyone, as mentioned in the comment.
I will use the official AOSP translation for it.

Comment (sr093906) [reply app/src/main/res/values-zh-rCN/strings.xml:39]: @iboalali They are not translated.

Comment (sr093906) [reply app/src/main/res/values-zh-rCN/strings.xml:39]: https://github.com/sr093906/sysnotifsnooze/blob/a08f32d5914335c85eec69338e288f8d10ec04b6/app/src/main/res/values-zh-rCN/strings.xml

Comment (iboalali) [reply app/src/main/res/values-zh-rCN/strings.xml:39]: Sorry 😓 looked at the wrong changes

**Pull request merged**